src/components/forms/VpnServerForm.js

```diff
@@ -0,0 +1,424 @@
+            invalidText={
+              !this.state.vpc || isNullOrEmptyString(this.state.vpc)
+                ? `Select a VPC.`
+                : `Select at least one security group.`
+            }
+            className={classNameModalCheck}
+          />
+          {/* certificate_crn */}
+          <IcseTextInput
+            id={this.props.data.name + "-vpn-server-certificate-crn"}
+            field="certificate_crn"
+            componentName="certificate_crn"
+            tooltip={{
+              content:
+                "Secrets Manager certificate unique identifier for VPN server",
+              align: "top-left",
+            }}
+            labelText="Secrets Manager Certificate CRN"
+            value={
+              this.state.certificate_crn === undefined
+                ? ""
+                : String(this.state.certificate_crn)
+            }
+            onChange={this.handleInputChange}
+            invalid={
+              isNullOrEmptyString(this.state.certificate_crn) ||
+              invalidCRNs(this.state.certificate_crn).invalid
+            }
+            invalidText={invalidCRNs(this.state.certificate_crn).invalidText}
+            className={classNameModalCheck}
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          {/* method toggle */}
+          <IcseSelect
+            formName={this.props.data.name + "-vpn-server-method"}
+            name="method"
+            labelText="Authentication Method"
+            groups={["Certificate", "Username"]}
+            value={titleCase(this.state.method)}
+            handleInputChange={this.handleInputChange}
+            className="fieldWidthSmaller"
+          />
+          {/* client_ca_crn */}
+          {this.state.method === "Certificate" && (
+            <IcseTextInput
+              id={this.props.data.name + "-vpn-server-client-ca-crn"}
+              field="client_ca_crn"
+              componentName="client_ca_crn"
+              labelText="Client Secrets Manager Certificate CRN"
+              value={
+                this.state.client_ca_crn === undefined
+                  ? ""
+                  : String(this.state.client_ca_crn)
+              }
+              onChange={this.handleInputChange}
+              invalid={
+                (this.state.method === "Certificate" &&
+                  isNullOrEmptyString(this.state.client_ca_crn)) ||
+                invalidCRNs(this.state.client_ca_crn).invalid
+              }
+              invalidText={invalidCRNs(this.state.client_ca_crn).invalidText}
+              className="fieldWidth"
+            />
+          )}
+          {/* client_ip_pool */}
+          <IcseTextInput
+            id={this.props.data.name + "-vpn-server-client-ip-pool"}
+            componentName="client_ip_pool"
+            name="client_ip_pool"
+            field="client_ip_pool"
+            value={this.state.client_ip_pool}
+            placeholder="x.x.x.x"
+            labelText="Client CIDR"
```

Comment:
label text => `Client CIDR Pool`
add tooltip 
content:
```
VPN client IPv4 address pool, expressed in CIDR format. The request must not overlap with any existing address prefixes in the VPC or any reserved address ranges.
```
link: https://cloud.ibm.com/docs/vpc?topic=vpc-vpn-client-to-site-overview